docs/source/app-dev/grpc/error-codes.rst

```diff
@@ -1,27 +1,26 @@
 .. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
+.. _error-codes:
```

Comment:
What does this do?

Reply:
So this page can be referenced from other places

Reply:
But since the file name is the same might not be needed. Let me check